package.json

```diff
@@ -48,7 +49,8 @@
     "verify:docs": "npm run build:site -- --verify",
     "build:site": "./bin/build-docs-site.sh",
     "mocha": "node packages/build/bin/run-mocha \"packages/*/DIST/test/**/*.js\" \"examples/*/DIST/test/**/*.js\" \"packages/cli/test/**/*.js\" \"packages/build/test/*/*.js\"",
-    "posttest": "npm run lint"
+    "posttest": "npm run lint",
+    "commitmsg": "commitlint -E GIT_PARAMS"
```

Comment:
just curious...does anywhere call this command?

Reply:
`husky` calls the `commitmsg` hook automatically when you do a `git commit -m "message"` ... which will lint your commit message locally instead of waiting for the CI